src/components/UI/AddBook.js

```diff
@@ -18,7 +18,14 @@ const AddBook = () => {
 
   const addBookHandler = (e) => {
     e.preventDefault();
-    dispatch(addBook({ id: uuidv4(), title, author }));
+    const book = {
+      item_id: uuidv4(),
+      title,
+      author,
+      category: 'Fiction',
+    };
+
+    dispatch(postBook(book));
```

Comment:
- Please, kindly ensure a user shouldn't be able to add a book if either the author or title input field is empty. see the image below::


![EMPTY](https://user-images.githubusercontent.com/59397159/185333745-c71ce27c-3000-46d9-b6d3-4f1a0695c86d.png)